.github/workflows/post-submit.yaml

```diff
@@ -19,7 +19,12 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
```

Comment:
I think it should be version 3 (see [here](https://github.com/actions/setup-go)).